src/package-lock.json

```diff
@@ -618,12 +618,6 @@
         }
       }
     },
-    "assert-plus": {
```

Comment:
Why has this commit got a package-lock change?

Reply:
I ran `npm audit fix`

Reply:
I'd probably put that in a separate commit with its own commit message explaining.